guides/common/modules/proc_example-restorecon-template.adoc

```diff
@@ -1,8 +1,9 @@
-:_mod-docs-content-type: REFERENCE
+:_mod-docs-content-type: PROCEDURE
 
 [id="Example_restorecon_Template_{context}"]
 = Example restorecon template
```

Comment:
```suggestion
[id="creating-a-job-template-example"]
= Creating a job template example
```

Same applies to the file name.

Alternative: "Creating a job template to restore SELinux context".

Reply:
I like the second one better, applied.